descqa/kcorrect_wrapper.py

```diff
@@ -0,0 +1,26 @@
+import kcorrect as kc
+import numpy as np
+from tqdm import tqdm
+def kcorrect(mags, magerrs, redshifts, bandshift, filters="des_filters.dat"):
+   kc.load_templates()
+   kc.load_filters(f=filters)
+   maggies=10.**(-0.4*mags)
+   maggies_ivar= 1./(maggies*0.4*np.log(10)*magerrs)**2 
+   kcorrect_arr = np.zeros_like(magerrs)
+   pbar = tqdm(total=len(mags))
+   for i, mag in enumerate(mags):
```

Comment:
use zip or stack `magerrs`, `maggies`, and `maggies_ivar`. Don't use `i` unless the index value itself it really needed.